vtysh/vtysh.c

```diff
@@ -4253,6 +4253,21 @@ DEFUN (vtysh_show_daemons,
 	return CMD_SUCCESS;
 }
 
+DEFPY (vtysh_exec_timeout,
+       vtysh_exec_timeout_cmd,
+       "[no$nono] exec-timeout ![(0-1000000)$tsecs]",
```

Comment:
nono or no? :)

Reply:
yeah, always get nervous about what the auto-generated variables are going to be...